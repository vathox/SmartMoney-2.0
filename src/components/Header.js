import React from 'react'
import {NavLink} from 'react-router-dom'

const Header = () => (
    <div>
        <h1>Smart Money 2.0</h1>
        <NavLink to="/" activeClassName="is-active" exact={true}>Dashboard</NavLink>
        <NavLink to="/create" activeClassName="is-active">Add new expense</NavLink>
        <NavLink to="/edit" activeClassName="is-active">Edit</NavLink>
        <NavLink to="/help" activeClassName="is-active">Help</NavLink>
    </div>
);

export default Header