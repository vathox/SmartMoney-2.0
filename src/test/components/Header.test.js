import React from 'react';
import Header from '../../components/Header'
import toJSON from 'enzyme-to-json'
import { shallow } from 'enzyme';

test('should render Header correctly', () => {
   const wrapper = shallow(<Header/>);
   expect(toJSON(wrapper)).toMatchSnapshot();
});