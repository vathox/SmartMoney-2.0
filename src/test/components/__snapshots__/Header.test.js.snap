// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <h1>
    SmartMoney 2.0
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/dashboard"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Add new expense
  </NavLink>
  <button>
    Logout
  </button>
</div>
`;
